ci-operator/step-registry/hypershift/kubevirt/run-e2e-external/hypershift-kubevirt-run-e2e-external-chain.yaml

```diff
@@ -16,20 +29,21 @@ chain:
         fi
       
         # run (pseudo) external infra e2e test
-        oc create ns guest-infra-ns
+        oc create ns guest-external-infra-ns
```

Comment:
this should be created on the infra cluster. I think the KUBECONFIG is supposed to be pointing to the mgmt cluster at this point. Maybe the kubeconfigs aren't being set accurately?

Reply:
the `KUBECONFIG` env var is indeed pointed to the mgmt cluster (which is itself an hypershift guest cluster).
I updated the namespace creation to be at the infra cluster, using the `INFRA_KUBECONFIG` env var.